src/app/website/company-home/feature-summary/feature-summary.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, OnInit } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/toPromise';
+import { ContentfulService } from '../../shared/contentful/contentful.service';
+
+
+@Component({
+  selector: 'tf-feature-summary',
+  templateUrl: './feature-summary.component.html',
+  styleUrls: ['./feature-summary.component.scss']
+})
+export class FeatureSummaryComponent implements OnInit {
+
+  contentData = [];
+  constructor(private http: Http,
+    private contentfulService: ContentfulService) { }
+
+
+
+  ngOnInit() {
+    this.getContents();
+  }
+
+  getContents() {
+    this.contentfulService.getFeatureSummaryData()
+      .subscribe(
+      (data: any) => {
+        for (const item of data.items) {
+          this.contentData.push(getValue(item));
+
+        }
+
+        function getValue(item) {
+          const sectionHeading = item.fields.sectionHeader;
+          const sectiDescription = item.fields.sectionDescription;
+          const advantages = [];
```

Comment:
What's the type of advantages. The whole point of using typescript it to give typesupport. Looks like you are bypassing everything and writing pure javascript.